tests/browser/batch_gift_entry/bge_custom_batch.robot

```diff
@@ -0,0 +1,66 @@
+*** Settings ***
+
+Resource        tests/NPSP.robot
+Library           DateTime
+Suite Setup     Open Test Browser
+Suite Teardown  Delete Records and Close Browser
+
+
+*** Test Cases ***
+
+BGE Batch With Custom Values
+    #Create a BGE batch with custom values (count/amount, new fields, new defaults, different settings)
+    [tags]  unstable
+    ${batch} =           Generate Random String
+    ${ns} =  Get NPSP Namespace Prefix
+    Select App Launcher Tab   Batch Gift Entry
+    Click BGE Button       New Batch
+    Fill BGE Form
+    ...                       Name=${batch}
+    ...                       Batch Description=This custom batch is created by Robot
+    ...                       Expected Count of Gifts=2
+    ...                       Expected Total Batch Amount=100
+    Click BGE Button        Next
+    Select Duellist Values    Opportunity    Available Fields    Donation Name
+    Click Duellist Button    Opportunity    Move selection to Selected Fields
+    ${xpath}    Get NPSP Locator    bge.duellist    Payment    Selected Fields    Check/Reference Number
+    Execute JavaScript    window.document.evaluate('${xpath}', document, null, XPathResult.FIRST_ORDERED_NODE_TYPE, null).singleNodeValue.scrollIntoView(true)
+    Select Duellist Values    Payment    Selected Fields    Check/Reference Number
+    Click Duellist Button    Payment    Move selection to Available Fields
+    Click BGE Button        Next
+    Select BGE Date Picker    Donation Date: Default Value
+    Change BGE month    Next Month
+    Pick BGE Date    20
+    Select Value From BGE DD    Method    Cash
+    Fill BGE Form
+    ...                       Donation Amount: Default Value=10
+    Click BGE Button        Next
+    Select Value From BGE DD    Donation Matching Behavior    Single Match - Only import a record if it matches a single existing record.
+    Click BGE Button    Show Advanced Options
+    Fill BGE Form
+    ...                       Batch Process Size=100
+    ...                       Number of Days from Donation Date=2   
+    ${xpath}    Get NPSP Locator    detail_page.section_header    Donation Matching
+    Execute JavaScript    window.document.evaluate('${xpath}', document, null, XPathResult.FIRST_ORDERED_NODE_TYPE, null).singleNodeValue.scrollIntoView(true)
```

Comment:
The Selenium library already has a `Scroll Element Into View` keyword -- but I'm guessing there's a reason you didn't use it @skristem?

Reply:
yes that was not working as I was not scrolling the page, instead of I was trying to scroll inside the modal on the page so I had to use execute javascript